### PR TITLE
Adding some addtional tests

### PR DESCRIPTION
**Pull request by nathan-weinberg**
Added tests for:
- `get_bugs_set()`
- `get_jira_set()`
- `has_blockers()`

### CODE REVIEW

File: test_functions.py [hunk 1]
```diff
@@ -18,6 +36,25 @@ def test_get_osp_version_func():
...
 
 
+def test_has_blockers():
+	mockers = {
+		'job1': {'bz': [123456]},
+		'job2': {'jira': ['RHOSINFRA-123']},
+		'job3': {'other': {'name': 'this is a test name'}},
+		'job4': {'bz': [0]},
+		'job5': {'jira': [0]},
+		'job6': {'other': [0]},
```
Comment (EliadCohen) [inline test_functions.py:46]:
```suggestion
		'job6': {'other': [0]},
		'job7': {'owners:' 'foo@bar.com'},
		'job8': {'bz': [123456], 'jira': ['RHOSINFRA-123'], 'owners:' 'foo@bar.com'},
		'job9': {}
```

File: test_functions.py [hunk 2]
```diff
@@ -18,6 +36,25 @@ def test_get_osp_version_func():
...
+		'job7': {}
```
Comment (EliadCohen) [inline test_functions.py:47]:
```suggestion
```

File: test_functions.py [hunk 3]
```diff
@@ -18,6 +36,25 @@ def test_get_osp_version_func():
...
+	}
+	assert has_blockers(mockers, 'job1') == True
+	assert has_blockers(mockers, 'job2') == True
+	assert has_blockers(mockers, 'job3') == True
+	assert has_blockers(mockers, 'job4') == False
+	assert has_blockers(mockers, 'job5') == False
+	assert has_blockers(mockers, 'job6') == False
+	assert has_blockers(mockers, 'job7') == False
```
Comment (EliadCohen) [inline test_functions.py:55]:
```suggestion
	assert has_blockers(mockers, 'job7') == True
	assert has_blockers(mockers, 'job8') == False
	assert has_blockers(mockers, 'job9') == True
```

**Comment by nathan-weinberg** [comment]
@EliadCohen b/c GitHub review formatting is burning my eyes - this is your full suggestion yes?

```python
def test_has_blockers():
	mockers = {
		'job1': {'bz': [123456]},
		'job2': {'jira': ['RHOSINFRA-123']},
		'job3': {'other': {'name': 'this is a test name'}},
		'job4': {'bz': [0]},
		'job5': {'jira': [0]},
		'job6': {'other': [0]},
		'job7': {'owners:' 'foo@bar.com'},
		'job8': {'owners:' 'foo@bar.com', 'bz': [123456], 'jira': ['RHOSINFRA-123']},
		'job9': {}
        assert has_blockers(mockers, 'job1') == True
	assert has_blockers(mockers, 'job2') == True
	assert has_blockers(mockers, 'job3') == True
	assert has_blockers(mockers, 'job4') == False
	assert has_blockers(mockers, 'job5') == False
	assert has_blockers(mockers, 'job6') == False
	assert has_blockers(mockers, 'job7') == True
	assert has_blockers(mockers, 'job8') == False
	assert has_blockers(mockers, 'job9') == True

**Comment by EliadCohen** [comment]
> @EliadCohen b/c GitHub review formatting is burning my eyes - this is your full suggestion yes?
> 
> ```python
> def test_has_blockers():
> 	mockers = {
> 		'job1': {'bz': [123456]},
> 		'job2': {'jira': ['RHOSINFRA-123']},
> 		'job3': {'other': {'name': 'this is a test name'}},
> 		'job4': {'bz': [0]},
> 		'job5': {'jira': [0]},
> 		'job6': {'other': [0]},
> 		'job7': {'owners:' 'foo@bar.com'},
> 		'job8': {'owners:' 'foo@bar.com', 'bz': [123456], 'jira': ['RHOSINFRA-123']},
> 		'job9': {}
>         assert has_blockers(mockers, 'job1') == True
> 	assert has_blockers(mockers, 'job2') == True
> 	assert has_blockers(mockers, 'job3') == True
> 	assert has_blockers(mockers, 'job4') == False
> 	assert has_blockers(mockers, 'job5') == False
> 	assert has_blockers(mockers, 'job6') == False
> 	assert has_blockers(mockers, 'job7') == False
> 	assert has_blockers(mockers, 'job8') == True
> 	assert has_blockers(mockers, 'job9') == False
> ```
Revised below:

```python
def test_has_blockers():
	mockers = {
		'job1': {'bz': [123456]},
		'job2': {'jira': ['RHOSINFRA-123']},
		'job3': {'other': {'name': 'this is a test name'}},
		'job4': {'bz': [0]},
		'job5': {'jira': [0]},
		'job6': {'other': [0]},
		'job7': {'owners': 'foo@bar.com'},
		'job8': {'owners': 'foo@bar.com', 'bz': [123456], 'jira': ['RHOSINFRA-123']},
		'job9': {}
	}
	assert has_blockers(mockers, 'job1') == True
	assert has_blockers(mockers, 'job2') == True
	assert has_blockers(mockers, 'job3') == True
	assert has_blockers(mockers, 'job4') == False
	assert has_blockers(mockers, 'job5') == False
	assert has_blockers(mockers, 'job6') == False
	assert has_blockers(mockers, 'job7') == False
	assert has_blockers(mockers, 'job8') == True
	assert has_blockers(mockers, 'job9') == False
```

**Pull request merged**